src/foam/u2/PopupView.js

```diff
@@ -53,7 +53,7 @@ foam.CLASS({
       var parent   = this.parentNode;
       var close    = function() {
         self.remove();
-        bg.remove();
+        //bg.remove();
```

Comment:
Rather than removing the background for all users of the PopupView, if you don't like the background, can we make it a configurable parameter of the PopupView?  Then your code can just say "PopupView.create({ modal: true })" or "fullscreen: true" or "overlay: true"

I think overlay: true works best.